tests/unit/ProvisionDispatcher.spec.js

```diff
@@ -44,12 +49,52 @@ function testDispatch() {
     });
 
     it('should be given all provisions when alone', done => {
-        clients[0] = new ProvisionDispatcher(zkConf);
+        const cbOnce = jsutil.once(done);
```

Comment:
I think done is already restricted to only being called once https://mochajs.org/#detects-multiple-calls-to-done

Reply:
Yes it will be called more than once in this case.